lib/localStorage.ts

```diff
@@ -1,18 +1,20 @@
 'use strict'
 
-const ObservableStore = require('.')
+import ObservableStore = require('.')
 
 class LocalStorageStore extends ObservableStore {
 
-  constructor (opts = {}) {
-    if (!global.localStorage) {
+  private _storageKey: string;
+
+  constructor (opts: { storageKey?: string } = {}) {
+    if (!(global as any).localStorage) {
```

Comment:
Should we do this here as well?
```suggestion
    if (!(global as unknown as Window).localStorage) {
```